semeio/reporter/reporter.py

```diff
@@ -0,0 +1,32 @@
+class Reporter:
+    def __init__(self, name):
+        self.name = name
+
+    def _get_suffix_for_value(self, value):
```

Comment:
Not so certain this is the best way of doing it..

The idea is that we could be able to separate write implementations for `dict`, `numpy`, `pd.DataFrame` etc. But it very quickly becomes a mess.. and really gold plating